frontend/src/lib/components/Support/SupportModal.tsx

```diff
@@ -63,6 +88,32 @@ export function SupportModal({ loggedIn = true }: { loggedIn?: boolean }): JSX.E
         },
     })
 
+    const {
+        setFilesToUpload: setFilesToUpload2,
+        filesToUpload: filesToUpload2,
+        uploading: uploading2,
+    } = useUploadFiles({
+        onUpload: (url, fileName) => {
+            setSendSupportRequestValue('message2', sendSupportRequest.message + `\n\nAttachment "${fileName}": ${url}`)
+        },
+        onError: (detail) => {
+            lemonToast.error(`Error uploading image: ${detail}`)
+        },
+    })
+
+    const {
+        setFilesToUpload: setFilesToUpload3,
+        filesToUpload: filesToUpload3,
+        uploading: uploading3,
+    } = useUploadFiles({
+        onUpload: (url, fileName) => {
+            setSendSupportRequestValue('message3', sendSupportRequest.message + `\n\nAttachment "${fileName}": ${url}`)
+        },
+        onError: (detail) => {
+            lemonToast.error(`Error uploading image: ${detail}`)
+        },
+    })
+
```

Comment:
can't you already upload several files, why this copy paste and let's not have a hardcoded limit of 3

Reply:
this is the important part: `..setSendSupportRequestValue('message3'..`
so that the uploaded file will be added into the correct field